pricing-app/src/app/admin/orders/orderItem.tsx

```diff
@@ -0,0 +1,49 @@
+import SelectionBox from "@/components/selectionBox";
+import { orderStatus } from "@/mock/data";
+import { Item, Options } from "@/types/type";
+import { FC } from "react";
+
+type Props = {
+  id: string;
+  createdAt: string;
+  updatedAt: string;
+  cost: number;
+  status: string;
+  onHandleUpdateOrder: (id: string) => void;
+  onChangeStatusOrder: (value: Options) => void;
+};
+
+const OrderItem: FC<Props> = ({
+  id,
+  createdAt,
+  updatedAt,
+  cost,
+  status,
+  onHandleUpdateOrder,
+  onChangeStatusOrder,
+}: Props) => {
+  return (
+    <div className="mb-4">
+      <div className="flex w-full justify-between" onClick={() => {}}>
+        <div className="">{`$${cost}`}</div>
```

Comment:
is this necessary?

Reply:
Yes, I want to show the cost of each order.